server/src/sessionManagement.mjs

```diff
@@ -140,6 +374,29 @@ function addCall(methodCall, params){
     }
 }
 
+function setOutputFormat(format){
```

Comment:
See comments earlier about this being multi-value.

Reply:
Multi value was not specified in the scope of this ticket, can create a ticket if we want it added.

Reply:
yep. np.